docs/docs-content/clusters/edge/architecture/two-node.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: "Two-Node Architecture"
+title: "Two-Node Architecture"
+description: "Learn about Palette Edge and the architecture used to support two-node edge clusters."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["edge", "architecture"]
+---
+
+Palette Edge allows you to provision a highly available (HA) cluster capable of withstanding any single node failure
+with only two nodes instead of three. Palette achieves this by sidestepping a critical limitation of etcd and Postgres
+as the backend storage through [Kine](https://github.com/k3s-io/kine).
```

Comment:
```suggestion
with only two nodes instead of three. Palette achieves this by sidestepping a critical limitation of etcd and Postgres
as the backend storage by using [Kine](https://github.com/k3s-io/kine).
```

Reply:
Maybe?